File: Pod/Tests/EmailFormatterTests.m
@import XCTest;

#import "EmailFormatter.h"

@interface EmailFormatterTests : XCTestCase

@end

@implementation EmailFormatterTests

- (void)testFormatString {
    EmailFormatter *formatter = [EmailFormatter new];
    XCTAssertEqualObjects(@"test@example.com", [formatter formatString:@"t,e,st@example.com" reverse:NO]);
}

@end

File: Pod/Tests/EmailFormatterTests.swift
import UIKit
import XCTest

class EmailFormatterTests: XCTestCase {
    func testFormatString() {
        let formatter = EmailFormatter()
        XCTAssertEqual("test@example.com", formatter.formatString("t,e,s,t@example.com", reverse: false))
    }
}
